### PR TITLE
Make the content/form order configurable

### PR DESCRIPTION
**Pull request by alex-mo**
This uses the variables provided by campaignion_layout.

### CODE REVIEW

File: templates/page.tpl.php [hunk 1]
```diff
@@ -73,6 +73,7 @@
  * @see template_process()
  * @see html.tpl.php
  */
+$form_below_main = $order_enabled && $order === 'below-main';
```
Comment (katzmo) [inline templates/page.tpl.php]: Does it have to be that complicated? I’d prefer a simple Boolean, e.g. `$reverse = TRUE` – or do you expect more kinds of layout variations? :flushed: 

If really needed, additional page variables should be set in the page preprocess function, not the template file (we've tried that, it wasn’t such a good idea). The code should also take into account that the variable(s) might *not* be set, e.g. `campaignion_layout` does not have to be enabled.

Related to https://github.com/moreonion/campaignion/pull/273#discussion_r609370054

**Review by katzmo — CHANGES_REQUESTED** [review]

File: templates/page.tpl.php (hunk 1 above)
Comment (alex-mo) [reply templates/page.tpl.php]: Haha, having the same discussion for `campaignion_layout`... I think it's not overly complicated. If we are confident enough, that we only need a boolean, fine with me to change it into a boolean. I do not expect more variations, but I would not rule out the possibility (just out of plain caution). That's why I chose a string.

Comment (alex-mo) [reply templates/page.tpl.php]: But granted, `$reverse = TRUE` does sound much simpler... :innocent:

Comment (katzmo) [reply templates/page.tpl.php]: Looks a lot clearer now, thanks!

Still, without `campaignion_layout` you’ll get a "Undefined variable" notice → I’d suggest to use `empty()` in the if conditions.

Comment (alex-mo) [reply templates/page.tpl.php]: Changed.

**Review by katzmo — APPROVED** [review]

**Pull request merged**